website/content/blog/2025-12-19-385-preview.md

```diff
@@ -0,0 +1,101 @@
+---
+title: `3.8.5` Release Preview 1 available
+date: "2025-12-19"
+excerpt: The road to `3.8.5` release has begun
+tags: meeting
+author: MonoGame Foundation
+image: /images/blog/cover/monogame_foundation.png
+ogImage: /images/blog/cover/monogame_foundation.png
+---
+
+## MonoGame 3.8.5 Release Preview 1 has been officially released
+
+![Celebration](images/announcement.gif)
+
+We are proud to announce the first of the Release Preview of `3.8.5` (**3.8.5-develop.1**), the road to release has officially begun.
+
+## Highlights
+
+### Base MonoGame fixes / improvements
+
+* Binaries release (for offline use)
+* NetStandard 2.1 support
+* iOS / Android policy update changes.
+* New RNG (Random) implementation.
+* Extended GamePad support, now up to 8 controllers.
+* New HSL/HSV handling.
```

Comment:
May need more context here....  like "Support for HLS and HSV color conversions."